lib/hipchat/room.rb

```diff
@@ -30,6 +30,8 @@ def get_room
         raise UnknownRoom,  "Unknown room: `#{room_id}'"
       when 401
         raise Unauthorized, "Access denied to room `#{room_id}'"
+      when 403
```

Comment:
Ah I see. On v1 you get 403, but on v2 [you get 429]( https://www.hipchat.com/docs/apiv2/rate_limiting). Can you factor that difference into the exception checking? (only raise the RateLimit if v1)